composer.json

```diff
@@ -11,18 +11,25 @@
     ],
     "autoload": {
         "psr-4": {
+            "EzSystems\\EzPlatformCoreBundle\\": "src/EzPlatformCoreBundle/bundle/",
             "EzSystems\\EzPlatformEncoreBundle\\": "src/EzPlatformEncoreBundle/bundle/"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "EzSystems\\Tests\\EzPlatformCoreBundle\\": "tests/EzPlatformCoreBundle/bundle/"
+        }
+    },
     "require": {
         "php": "^7.1",
         "symfony/symfony": "^3.4"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.15",
-        "phpunit/phpunit": "^6.4"
+        "phpunit/phpunit": "^8.3.5"
     },
     "scripts": {
+        "test": "@php ./vendor/bin/phpunit -v -c phpunit.xml",
```

Comment:
Composer will push the `vendor/bin` folder to the top of the `$PATH` variable before executing scripts so the `phpunit ` form `vendor/bin` will be prioritized over anything installed globally anyway. More info: https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands

Reply:
> Composer will push the `vendor/bin` folder to the top of the `$PATH` variable before executing scripts so the `phpunit ` form `vendor/bin` will be prioritized over anything installed globally anyway. More info: https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands

I can try this again, but AFAIR last week tests on Travis failed w/o that. I suspected that maybe Travis ships its own version of phpunit.

Reply:
> > Composer will push the `vendor/bin` folder to the top of the `$PATH` variable before executing scripts so the `phpunit ` form `vendor/bin` will be prioritized over anything installed globally anyway. More info: https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands
> 
> I can try this again, but AFAIR last week tests on Travis failed w/o that. I suspected that maybe Travis ships its own version of phpunit.

Nvm, must've been PEBKAC on my side. It works, thanks!